job_executor/worker/steps/dataset_validator.py

```diff
@@ -8,7 +8,7 @@
 
 
 logger = logging.getLogger()
-INPUT_DIR = Path(environment.get('INPUT_DIR'))
+INPUT_DIR = Path(environment.get('INPUT_DIR')) / 'archive'
```

Comment:
Can we name this const ARCHIVE_DIR / INPUT_ARCHIVE_DIR and refer to it in line 26?:
```py
input_directory=str(INPUT_ARCHIVE_DIR),
```